File: my_solution.js
function(s) {
    let str = s.trim().split(' ')
    console.log(str)
    
    let reversed = []
    for(let i = str.length - 1; i >= 0; i--){
        if(str[i].length > 0){
            reversed.push(str[i])
        }
    }
    
    return reversed.join(' ')
};

File: my_solution.py
def reverseWords(self, s):
        str = s.strip().split(' ')
        
        reversed = []
        for i in range(len(str) - 1, -1, -1):
            if len(str[i]) > 0:
                reversed.append(str[i])
        
        return ' '.join(reversed)
